docs/fitting_types.rst

```diff
@@ -385,6 +385,28 @@ are added squared, ready to be minimized. Unlike in the above example, the
 
   Do not cite the overall :math:`R^2` given by :mod:`symfit`.
 
+Fitting moredimensional datasets
+--------------------------------
+Imagine that you have now a specific property over a grid, like a temperature,
+so you have a three-dimensional data set. To fit a function with two parameters
+to this data set, you can use :mod:`symfit`.
+
+At this example, we made a polynomial function with two coefficients,
+representing two terms of mixed order in x and y::
+
+  x, y, z = variables('x, y, z')
+  c1, c2 = parameters('c1, c2')
+  model_dict = {z: Poly( {(1, 2): c1, (4, 5): c2}, x ,y)}
```

Comment:
Most readers won't know this `Poly` syntax. It would be good to either print the model made, or write out the polynomial above.

Reply:
I would add the following line at the end of the first codeblock:
`#printed version: z(x, y; c1, c2) = Poly(c2*x**4*y**5 + c1*x*y**2, x, y, domain='ZZ[c1,c2]')`

Reply:
Or:
```python3
# >>> print(model)
# z(x, y; c1, c2) = Poly(c2*x**4*y**5 + c1*x*y**2, x, y, domain='ZZ[c1,c2]')
```
Or put the mathematical form (:math:`c2*x^4*y^5 + c1*x*y^2`) near where you say "we made a polynomial function"

Reply:
I'll add the mathematical form

Reply:
I would keep the `Poly` object because it is far more powerful since it can generate arbitrary polynomials. I think it would be nice if you could elaborate a bit on the math form. If you write using `:math:` in the docs which polynomial you are making, then it will also be clearer in the code even without a print.